modules/generator-primer-module/app/templates/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "0.1.0",
+  "name": "<%= module %>",
+  "description": "<%= description %>",
+  "homepage": "http://primercss.io/",
+  "author": "GitHub, Inc.",
+  "license": "MIT",
+  "style": "index.scss",
+  "main": "build/index.js",
```

Comment:
Looks like files are missing, every module should have:
```
"files": [
    "index.scss",
    "lib",
    "build"
  ],
```

And some have `docs` too, such as `primer-utilities`.

Reply:
I'm fine with blacklisting, but will let @jonrohan comment on that :)

Reply:
if it works :+1::skin-tone-2: I recall struggling with that trying to figure out how to publish files that are built but not checked into the repository